app/dturn/dturn.js

```diff
@@ -121,6 +122,9 @@ function insertStatementId(discussionId, userId, statementId) {
       },
     },
   })
+
+  Discussions[discussionId].updates({ participants: Object.keys(Discussions[discussionId].Uitems).length })
```

Comment:
I think something will need to be added to Deliberations[] to keep track of the number of participants, and the lastRound, so that we only broadcast an update when there's a change in one of those values.